src/stages/normalize/patchers/FunctionApplicationPatcher.js

```diff
@@ -82,7 +82,17 @@ export default class FunctionApplicationPatcher extends NodePatcher {
     let { args } = this;
     let lastArg = args[args.length - 1];
     if (lastArg.isMultiline()) {
-      this.insert(this.contentEnd, `\n${this.getIndent()})`);
+      /**
+       * in case the next non-whitespace token is `.` we have to put the ) on it's line, so we lookahead.
+       */
+      let nextInterestingIdx = this.indexOfSourceTokenBetweenSourceIndicesMatching(
+        this.contentEnd, this.parent.contentEnd, t => !this.slice(t.start, t.end).match(/^\s*$/));
```

Comment:
I think the `isSemanticToken` method should handle that regex code (and it's often used by default). I think you may be able to do something similar to `getFollowingOpenParenToken` in #773 that I just put up.